localstack/aws/api/lambda_/__init__.py

```diff
@@ -233,6 +233,7 @@ class Runtime(str):
     python3_11 = "python3.11"
     nodejs20_x = "nodejs20.x"
     provided_al2023 = "provided.al2023"
+    nodejs20_x = "nodejs20.x"
```

Comment:
Looks like this is a duplicate of line 234.

Reply:
Good catch, updated in [`a5dec38`](https://github.com/localstack/localstack/pull/9659/commits/a5dec3830c0319414d7dacc912734825c4e0c947)